control/lti.py

```diff
@@ -526,22 +539,27 @@ def freqresp(sys, omega, squeeze=True):
     ----------
     sys: StateSpace or TransferFunction
         Linear system
-    omega : float or array_like
+    omega : float or 1D array_like
         A list of frequencies in radians/sec at which the system should be
         evaluated. The list can be either a python list or a numpy array
         and will be sorted before evaluation.
-    squeeze : bool, optional (default=True)
-        If True and `sys` is single input, single output (SISO), returns
-        1D array rather than a 3D array.
+    squeeze : bool, optional
+        If squeeze=True, remove single-dimensional entries from the shape of
+        the output even if the system is not SISO. If squeeze=False, keep all
+        indices (output, input and, if omega is array_like, frequency) even if
+        the system is SISO. The default value can be set using
+        config.defaults['control.squeeze_frequency_response'].
 
     Returns
     -------
-    mag : (p, m, len(omega)) ndarray or (len(omega),) ndarray 
+    mag : ndarray
         The magnitude (absolute value, not dB or log10) of the system
-        frequency response. Array is ``(len(omega), )`` if and only if system 
-        is SISO and ``squeeze=True``.
-
-    phase : (p, m, len(omega)) ndarray or (len(omega),) ndarray 
+        frequency response.  If the system is SISO and squeeze is not True,
+        the array is 1D, indexed by frequency.  If the system is not SISO or
+        squeeze is False, the array is 3D, indexed by the output, input, and
+        frequency.  If ``squeeze`` is True then single-dimensional axes are
+        removed.
+    phase : ndarray
```

Comment:
In ‘notes’ on line 575ish below, this function is actually a wrapper for :meth:’lti.frequency_response’

Reply:
Because of the way the documentation is built, the `LTI` class does not show up, so the documentation appears in the StateSpace and TransferFunction classes.  Leaving unchanged.